sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/example/BeamSqlExample.java

```diff
@@ -86,7 +86,10 @@ Void apply(Row input) {
     // Case 2. run the query with BeamSql.query over result PCollection of case 1.
     PCollection<Row> outputStream2 =
         PCollectionTuple.of(new TupleTag<>("CASE1_RESULT"), outputStream)
-            .apply(BeamSql.queryMulti("select c2, sum(c3) from CASE1_RESULT group by c2"));
+            .apply(
+                BeamSql
+                    .query("select c2, sum(c3) from CASE1_RESULT group by c2")
+                    .toPTransform());
```

Comment:
Yeah, the new syntax definitely looks like a regression in usability.

Reply:
Updated, kept the old syntax